docs/config.rst

```diff
@@ -41,3 +41,20 @@ An environment variable like ``REACT_APP_FOO`` controls the setting ``FOO``.
     :default: ``https://${OIDC_DOMAIN}/userinfo``
 
    The audience for the access token that is generated by Auth0.
+
+
+.. envvar:: REACT_APP_REFRESH_AUTH_PERIOD_SECONDS
+
+    :default: 300
+
+    How often to issue a silent authentication refresh. Technically, when you're logged in,
+    an infinite loop is run that refreshes your access token forever. And this number is
+    number of seconds to sleep between each check.
+
+.. envvar:: REACT_APP_REFRESH_AUTH_PREEMPTIVE_SECONDS
+
+    :default: 900
+
+    How soon *before* the session expires do we perform a silent authentication to keep it fresh.
+    If you set this really low (or even to zero) the risk is that the access token has expired
+    by the time it's needed for an API endpoint call.
```

Comment:
I would reword this last sentence along the lines of
```
If you set this lower than `REACT_APP_REFRESH_AUTH_PERIOD_SECONDS` then the access token might have expired before it was detected that it needed a refresh.
```

Reply:
I looked at it and I think the previous sentence was easier to read. 
Your suggestion makes it feel like there are only two single things happening. The check if the access token has expired and the number of seconds before the access token expires. 

What it actually is is that the REACT_APP_REFRESH_AUTH_PERIOD_SECONDS is used all the time, repeatedly. 99% of the time nothing happens. But 1% of the time the business logic deems the access token expired (or about to expire) and thus needs to be refreshed.